### ISSUE TITLE
Feature request: Option to use existing audio files instead of recording new

### ISSUE THREAD

**Issue by notrealdan** [BlitzKraig/fvtt-VoiceActor#2]
Would it be possible to have the option to point to an existing audio file, in addition to the existing recording function?  Just a thought that came to me while I was playing with this module.  Thanks!

**Comment by BlitzKraig** [comment]
Hi notrealdan,
Thanks for the feedback!
Please find below a conversation I had with a user on Discord about this today, explaining my current position on this.
There is also a short guide on how to achieve this with the current module.

> **Redacted at 12:46**
> Blitz great new version of VoiceActor. Adding it to the journal entry it's very useful.
> Did you though of adding an import button? This way the voice could be recorded and manipulated before been put in the journal entry?
> **BlitzToday at 13:17**
> Hi, thanks! Yes, I've had a think about this, still deciding whether or not this is out of scope. I want to ensure the module doesn't get too complicated. However, using Foundrys filepicker, I can see an argument for adding it. I'll mull it over a bit more
> In the meantime, you could get the actors ID, then drop a `{id}.ogg` file directly in the VoiceActor directory that gets created
> **BlitzToday at 13:41**
> For the sake of completeness: 
> Select a token, press F12 to open devtools, go to console, type `canvas.tokens.controlled[0].actor.data._id`
> This will print the actors ID to the console. (e.g. "SGt9eiJi12N70wO3")
> You can then record some audio outside of foundry, do whatever you want to it, and save it as `SGt9eiJi12N70wO3.ogg` inside `VoiceActor/` in your userdata folder.
> Now when you refresh, you should be able to open the actor and playback that clip
> Similarly for journals:
> Get the id by typing `game.journal.getName("Name of your journal entry").id`
> Same process for the rest of it, except you should save the file in `VoiceActor/Journal/`

As mentioned in the linked conversation, I'm still deciding whether or not to do this, though with 2 reports now it seems like it would be a useful feature for multiple people, so I'll keep that in mind.

Thanks again for the feedback!

**Comment by notrealdan** [comment]
Thanks for the detailed response!  I was thinking about this some more, and I suppose that if this other feature request of your other module were implemented, I wouldn't have as much desire for this request anymore, especially if I could put such sound links say, in an NPC's description text:
https://github.com/BlitzKraig/fvtt-SoundBoard/issues/25

If I had to vote for one of these features to get attention, it would be that one for SoundBoard, in case it matters.  It would make the SoundBoard quite a bit more useful in my eyes.  Thanks again!
